app/controllers/admin/snap_applications_controller.rb

```diff
@@ -17,5 +17,23 @@ class SnapApplicationsController < Admin::ApplicationController
 
     # See https://administrate-prototype.herokuapp.com/customizing_controller_actions
     # for more information
+    #
+    def resend_fax
+      application = SnapApplication.find(params[:id])
+      application.exports.faxed.succeeded.update_all(status: :failed, metadata:
```

Comment:
super minor style thing: I think this would be more readable as

```
application.exports.faxed.succeeded.update_all(
   status: :failed, 
   metadata:  "Reset by admin",
)
```